dbus/mako.c

```diff
@@ -14,82 +14,84 @@
 static const char *service_path = "/fr/emersion/Mako";
 static const char *service_interface = "fr.emersion.Mako";
 
-static int handle_dismiss_all_notifications(sd_bus_message *msg, void *data,
+static int handle_dismiss(sd_bus_message *msg, void *data,
 		sd_bus_error *ret_error) {
 	struct mako_state *state = data;
 
-	close_all_notifications(state, MAKO_NOTIFICATION_CLOSE_DISMISSED);
-	struct mako_surface *surface;
-
-	wl_list_for_each(surface, &state->surfaces, link) {
-		set_dirty(surface);
+	if (wl_list_empty(&state->notifications)) {
```

Comment:
I think I'd prefer to remove this check, so that the function behaves the same when there are notifications and when there are none.